src/App.js

```diff
@@ -16,38 +17,36 @@ import React, { Component } from 'react'
 export default class App extends Component {
   state = {}
 
-  handleHornedChange = (e) => {
+  handleAnimalChange = (e) => {
     this.setState({choice: e.target.value})
   }
 
+  handleHornChange = (e) => {
+    this.setState({horns: e.target.value})
+  }
+
   render() {
     const hornedAnimals = images
-        let filteredHorns = hornedAnimals
+    let filteredHorns = hornedAnimals
+
+    if (this.state.choice) {
+        filteredHorns = filteredHorns.filter( animal => animal.keyword === this.state.choice)
+    }
+
+    if (this.state.horns) {
+      filteredHorns = filteredHorns.filter( animal => animal.horns === +(this.state.horns))
+  }
 
-        if (this.state.choice) {
-            filteredHorns = hornedAnimals.filter( animal => animal.keyword === this.state.choice)
-        }
+    const animalTypeOptions = Array.from(new Set(hornedAnimals.map(animal => animal.keyword)));
+    const animalHornsOptions = Array.from(new Set(hornedAnimals.map(animal => animal.horns)))
```

Comment:
just a reminder, would like to see more commits for each lab.   Commit as often as possible!